java/src/problems/problem2/ExamMarksFactory.java

```diff
@@ -7,6 +7,6 @@
 
     @Override
     public List<ExamMarks> create() {
-        return Arrays.asList(new ToniImp(), new JoseLlorensImp());
+        return Arrays.asList(new ToniImp(), new JoseLlorensImp(), new KevinImp());
```

Comment:
You need have a pull from original repository, because this line has changed!

Download the last versión of the repository!